--- basic/vendor/yiisoft/yii2/web/Request.php	2022-02-11 13:12:40.000000000 +0000
+++ target/vendor/yiisoft/yii2/web/Request_session.php	2022-07-18 17:13:11.000000000 +0000
@@ -133,7 +133,7 @@
      * in session under the name of [[csrfParam]]. Note that while storing CSRF tokens in session increases
      * security, it requires starting a session for every page, which will degrade your site performance.
      */
-    public $enableCsrfCookie = true;
+    public $enableCsrfCookie = false;
     /**
      * @var bool whether cookies should be validated to ensure they are not tampered. Defaults to true.
      */
